src/main/resources/templates/client/fragments/header.html

```diff
@@ -5,7 +5,7 @@
 <body>
 <div th:fragment="header(activeTab, parentScreen)">
     <nav class="navbar navbar-default navbar-red">
-        <a th:unless="${parentScreen} == ''"  th:href="${parentScreen}" class="btn back" type="button">
+        <a th:unless="${parentScreen} == ''"  th:href="${parentScreen}" class="btn back">
```

Comment:
Nevim, jestli autor nechtel pouzit `role="button"`, ale myslim, ze je to v tomto pripade zbytecne.

Reply:
Dává smysl, změnil jsem.